src/fe/fe_base.C

```diff
@@ -753,14 +811,13 @@ void FEGenericBase<Real>::compute_dual_shape_coeffs ()
       for (auto j : index_range(phi))
         A(i,j) += JxW[qp]*phi[i][qp]*phi[j][qp];
     }
-
   // dual_coeff = A^-1*D
   for (auto j : index_range(phi))
   {
     DenseVector<Real> Dcol(sz), coeffcol(sz);
-    for (auto i : index_range(phi))
-      Dcol(i) = D(i, j);
-    A.cholesky_solve(Dcol, coeffcol);
+    Dcol.zero(); coeffcol.zero();
+    Dcol(j) = D(j, j);
+    A.lu_solve(Dcol, coeffcol);
```

Comment:
You've changed from Cholesky solve to LU here? Since A is symmetric, Cholesky should do fewer operations and have better round-off error characteristics than LU. It may be that there's some issue with Cholesky if you are getting significantly different results by making this change...

Reply:
I had this tested with both LU and Cholesky, which does not seem to give different results. Since Cholesky is more efficient and more accurate, I will change it back. Thanks for the detail comparison of these two!